src/sense_eeg.cpp

```diff
@@ -63,20 +63,30 @@ void publish_data(void* eeg_publisher)
     delete buffer;
 }
 
-GtecEEGPublisher::GtecEEGPublisher() : Node("gtec_eeg_publisher")
-{
-    num_channels = 1;
-    sampling_rate = 256;
-    serial_num = "UR-2017.06.12";
+GtecEEGPublisher::GtecEEGPublisher() : Node("gtec_eeg_publisher"),
 
+    num_channels(declare_parameter<int>("num_channels", 1)),
+    sampling_rate(declare_parameter<double>("sampling_rate", 256.0)),
```

Comment:
I just tested this again as there's been a few changes today. All working well. One small change, we need to declare this as an `int` here too.
```suggestion
    sampling_rate(declare_parameter<int>("sampling_rate", 256.0)),
```

Reply:
Thank you Stephen! Overlooked that one